src/framework/mocha/Utilities/MOFunctionArgument.m

```diff
@@ -905,9 +905,15 @@ + (BOOL)toJSValue:(JSValueRef *)value inContext:(JSContextRef)ctx typeEncoding:(
                 return YES;
             }
             
-            // Convert to NSString and then to JavaScript string
-            NSString *name = [NSString stringWithUTF8String:charPtr];
-            JSStringRef    jsName = JSStringCreateWithCFString((__bridge CFStringRef)name);
+            CFStringRef name = CFStringCreateWithCString(NULL, charPtr, kCFStringEncodingUTF8);
+            if (name == NULL) {
+                // for some reason, in some case, the string can be nil. Sight.
```

Comment:
"Sight" ?

Reply:
yeah "Sight" because it shouldn't be possible but it happens sometimes. Another mystery of CocoaScript.

Reply:
I'm still struggling to understand what's the right word here. I blame English 😉 

"Slight"? As in "there's a slight chance of this happening"?
"Psyche"? Cocoascript is trying to psyche you out.

Reply:
sight as in sigh 🙈